### PR TITLE
RUMM-3252 Roku logs documentation

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Adds the documentation for Logs collection using Datadog's Roku library

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/log_collection/roku.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://github.com/DataDog/dd-sdk-roku
+[2]: /account_management/api-app-keys/#client-tokens
+[3]: /account_management/api-app-keys/#api-keys
+[4]: /logs/processing/attributes_naming_convention/
+[5]: /tagging/
+[6]: /real_user_monitoring/roku/?tab=us
+[7]: [/real_user_monitoring/roku/?tab=us](https://github.com/DataDog/dd-sdk-roku/releases)
```
Comment (plousada) [inline content/en/logs/log_collection/roku.md:197]:
```suggestion
[7]: https://github.com/DataDog/dd-sdk-roku/releases
```
Should be this instead, no?

File: content/en/logs/log_collection/roku.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+1. Add the dependency to your project using `ROPM`, or download the [latest zip archive][7] and extract it to your project.
+
+    ```shell
+    ropm install datadog-roku
+    ```
+
+2. Initialize the library with your [Datadog client token][2] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Roku RUM Collection][6] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-roku` library as they would be exposed client-side in the Roku channel's package. For more information about setting up a client token, see the [client token documentation][2].
+
+    {{< site-region region="us" >}}
+    ```brightscript
```
Comment (plousada) [inline content/en/logs/log_collection/roku.md:40]: This syntax highlight doesn't seem to be working in the docs preview.

File: content/en/logs/log_collection/roku.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+1. Add the dependency to your project using `ROPM`, or download the [latest zip archive][7] and extract it to your project.
+
+    ```shell
+    ropm install datadog-roku
+    ```
+
+2. Initialize the library with your [Datadog client token][2] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Roku RUM Collection][6] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-roku` library as they would be exposed client-side in the Roku channel's package. For more information about setting up a client token, see the [client token documentation][2].
+
+    {{< site-region region="us" >}}
```
Comment (plousada) [inline content/en/logs/log_collection/roku.md:39]: Site selector doesn't seem to be working in docs preview.

File: content/en/logs/log_collection/roku.md (hunk 1 above)
Comment (0xnm) [inline content/en/logs/log_collection/roku.md:197]: seems like a wrong link here

**Review by plousada — APPROVED** [review]

**Review by alai97 — APPROVED** [review]
Looks great, thanks for opening this PR!

**Pull request merged**